2/introduction.md

```diff
@@ -1,6 +1,25 @@
-* What does address signify in a p2p system? Why is it imp?
-* What is multiadrs in libp2p (use "multi" + "adrs")
-* examples of it, how to construct it? bla bla
+Peer Identity in Libp2p and the Future
+======================================
 
-## Use of Multiadrs in our awesome-p2p-chat
-* Will help us exaclty locate the peer, such that we can send messages through transport.
\ No newline at end of file
+### Peer Identity
+Libp2p is intended to be a future-proof, modular networking protocol where you can pick the features you want in your stack.  The best example of how both of these goals are accomplished in libp2p is through the use of the multiaddress.  Before we dig into multiaddresses and how they future-proof libp2p efficiently, we have to talk about identity and what it means in libp2p.  
+
+In any sort of networking environment, you need a way to uniquely identify the nodes on the network.  This is the idea of peer identity, what you might call an address, a unique piece of data a node uses to identify itself to the world.  IP addresses or MAC addresses are good examples of an address used to identify servers and devices within networks of various types.
```

Comment:
This explanation seems ok. But can we add something more relatable to day-to-day use? Like a mobile number can be used as a peer identity.